src/AzureClient/Magic/SubmitMagic.cs

```diff
@@ -51,6 +51,8 @@ public SubmitMagic(IAzureClient azureClient, ILogger<SubmitMagic> logger)
 
                         - `{AzureSubmissionContext.ParameterNameJobName}=<string>`: Friendly name to identify this job. If not specified,
                         the Q# operation or function name will be used as the job name.
+                        - `{AzureSubmissionContext.ParameterNameJobParams}=< ImmutableDictionary<string, string>>`: Provider-specific job parameters
```

Comment:
Same here.

Reply:
Please see reply to the ExecuteMagic.cs comment.